docs/quick_start.md

```diff
@@ -195,7 +186,7 @@ const StyleDictionary = require('style-dictionary').extend('config.json');
 StyleDictionary.buildAllPlatforms();
 ```
 
-The `.extend()` method is an overloaded method that can also take an object with the configuration in the same format as a config.json file.
+The `.extend()` method is an overloaded method that can also take a [configuration](config) object.
```

Comment:
Missing ".md"

Reply:
weirdly it works without the md. I was putting them back in for clarity / consistency.